kernel/interrupts/src/aarch64/table.s

```diff
@@ -77,9 +77,10 @@ __exception_vector_start:
 .org 0x200
     CALL_WITH_CONTEXT current_elx_synchronous
 .org 0x280
+    msr daifclr, #1
```

Comment:
> is this needed? on x86 interrupts are disabled automatically by the hardware when entering an interrupt handler. Does aarch64 behave differently?
> 
> And if this is required, how do interrupts get re-enabled? Does the hardware do that automatically? Seems weird that it would do it one way but not the other.

I think you interpreted this instruction as "disable some interrupts", whereas it enables fast interrupts.

Reply:
lol, yep, i did certainly misinterpret it. Thanks for clarifying. Again, I'd recommend putting what you just wrote directly in a comment here so that information is preserved for future readers.